src/components/MoneyRequestReportView/MoneyRequestReportActionsList.tsx

```diff
@@ -144,6 +144,18 @@ function MoneyRequestReportActionsList({
 
     const canPerformWriteAction = canUserPerformWriteAction(report);
 
+    const isMissingReportActions = useMemo(() => {
+        return (
+            reportActions.filter(
+                (reportAction) =>
+                    (isOffline || isDeletedParentAction(reportAction) || reportAction.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE || reportAction.errors) &&
+                    shouldReportActionBeVisible(reportAction, reportAction.reportActionID, canPerformWriteAction),
+            )?.length === 0
+        );
+    }, [canPerformWriteAction, isOffline, reportActions]);
```

Comment:
This logic is pretty similar to `visibleReportActions`. Merge them into one `useMemo`